tests/regression_tests/mgxs_library_distribcell/test.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, *args, **kwargs):
         self.mgxs_lib.by_nuclide = False
 
         # Test all MGXS types
-        self.mgxs_lib.mgxs_types = openmc.mgxs.MGXS_TYPES + \
+        self.mgxs_lib.mgxs_types = openmc.mgxs.MGXS_TYPES[:-1] + \
```

Comment:
Would prefer a loop that excludes the current mgxs explicitly. If someone adds a mgxs after, they'd have to update the -1 to -2.